### DISCUSSION TITLE
URL Links from Objects

### DISCUSSION THREAD

**Discussion by emilianodelau** [mozilla/hubs#2979] [General Questions & Discussion] [answered]
Can I associate a URL with an image or object of my choosing? I'd like to link an image or object to a URL.

**Comment by Letitbe133** [comment]
You can do that in Spoke by inserting a **link** element providing the url to the media you want to display.
you can also do that directly from the room itself by clicking on the **magic wand** on top of the screen and then on **Custom URL or File** on the top right corner. Notice in this case that you will have to **pin** the link for it to remain in the room when you leave. Hope that helps

**Reply by davegoopot** [reply]
I'm not sure whether that was the question. The answer @Letitbe133 gave was to "How do I load media into Hubs from a URL?". I read the question as "How do I link out of Hubs from an element to a URL"?

See also my question: #2981

**Reply by Letitbe133** [reply]
My bad @davegoopot , I may have misread the question

**Reply by emilianodelau** [reply]
@davegoopot is correct. My question was how to take any object (image or 3D model) and associate a URL to it so that when someone clicks on the object in Hubs it opens the destination URL in a new tab?

**Comment by brianpeiris** [comment, marked as answer]
This is currently not possible, but it sounds like a good idea and it should be relatively easy for us to implement. I'd suggest creating a feature request through this link:

https://github.com/mozilla/hubs/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=

**Reply by davegoopot** [reply]
@emilianodelau has done the honours: ~~#2987~~ UPDATE: this is now being tracked in issue #2933
